client/src/components/profileImage.vue

```diff
@@ -4,7 +4,7 @@
       <img
         :src="imageSrc + user.image"
         class="user-profile-image rounded-circle"
-        style="width: 70px; height: 70px"
+        :style="{ width: width || '70px', height: width || '70px', objectFit: 'cover' }"
```

Comment:
Shouldn't the width be static?
```suggestion
        :style="{ width: '70px', height: '70px', objectFit: 'cover' }"
```

Reply:
@zaelgohary no it should be configurable from outside

Reply:
fixed
![image](https://github.com/codescalers/CSHR/assets/10484295/9669bb5b-5478-41d9-a8e2-8ccafa556877)